src/test/java/io/github/bonigarcia/wdm/test/DriverManagerTypeTest.java

```diff
@@ -50,6 +50,7 @@ public void test() {
     public static Collection<Object[]> data() {
         return asList(new Object[][] {
                 { "org.openqa.selenium.chrome.ChromeDriver", DriverManagerType.CHROME },
+                { "org.openqa.selenium.chromium.ChromiumDriver", DriverManagerType.CHROMIUM },
```

Comment:
AFAIK this class does not exist. Chromium is used with the class `org.openqa.selenium.chrome.ChromeDriver` and changing the Chromium binary path with options.

Reply:
Is a better approach to stay with the same class?

Reply:
I think so.